src/Silex/Provider/MonologServiceProvider.php

```diff
@@ -77,14 +98,19 @@ public function register(Container $app)
             return new LogListener($app['logger'], $app['monolog.exception.logger_filter']);
         };
 
-        $app['monolog.name'] = 'myapp';
+        $app['monolog.name'] = 'app';
         $app['monolog.bubble'] = true;
         $app['monolog.permission'] = null;
         $app['monolog.exception.logger_filter'] = null;
+        $app['monolog.logfile'] = null;
     }
 
     public function boot(Application $app)
     {
+        if (!$app['debug']) {
+            ErrorHandler::register($app['monolog']);
+        }
```

Comment:
Not sure why I did that (I extracted the code from an old project). Let's remove the condition and see if that works for everyone.